src/MongoDB.Driver.Legacy/MongoServerSettings.cs

```diff
@@ -96,7 +96,7 @@ public MongoServerSettings()
             _readPreference = ReadPreference.Primary;
             _replicaSetName = null;
             _retryReads = true;
-            _retryWrites = false;
+            _retryWrites = true;
```

Comment:
Also update public API documentation to indicate the new default value.

Also, @rstam, is there any place in the reference documentation where we add information about upgrading, similar to http://mongodb.github.io/mongo-java-driver/3.10/upgrading/.  I couldn't find anything, but this would be something I'd add.  If not, maybe add to the `what_is_new` page for the 2.9 release?

Reply:
Done